### ISSUE TITLE
Uncoverable statements in XS

### ISSUE THREAD

**Issue by Tux** [pjcj/Devel--Cover#111]
I'd like to have means to tag XS lines as uncoverable. Currently neither <code>/&#x2a; uncovered &#x2a;/</code> nor <code>/&#x2a; uncoverable statement &#x2a;/</code> achieve that goal. I prefer <code>/&#x2a; uncovered &#x2a;/</code>, as it is shorter. Best would be to also allow a reason.<br />
<code>if (unlikely_expression)</code>
<code>&nbsp; &nbsp; warn ("Unlikely event did occur"); /&#x2a; uncovered: out of memory? &#x2a;/</code>

**Comment by pjcj** [comment]
Agreed.  Something to mirror the possibilities for Perl code.  I will see what can be done.

Thanks!
